AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -174,7 +174,7 @@ + (void)generateTestCrash {
   @synchronized([MSCrashes sharedInstance]) {
     if ([[MSCrashes sharedInstance] canBeUsed]) {
       if ([MSUtility currentAppEnvironment] == MSEnvironmentAppStore) {
-          MSLogWarning([MSCrashes logTag], @"GenerateTestCrash was just called in an App Store environment. The call will be ignored");
+        MSLogWarning([MSCrashes logTag], @"GenerateTestCrash was just called in an App Store environment. The call will be ignored");
```

Comment:
We're missing a `.` inside the comment. Not technically part of this PR but let's fix it now.